spec/std/mutex_spec.cr

```diff
@@ -26,9 +26,9 @@ describe Mutex do
     mutex = Mutex.new
 
     a = 1
-    two = true
-    three = true
-    four = true
+    two = false
+    three = false
+    four = false
```

Comment:
Why?

Reply:
Because we eventually assert that they'll be `true`, so I wanted to also make sure the variables were changed —meaning that the fibers have run. We could even initialize them to `nil`.

Reply:
I guess because the assertions at the end are `should be_true` so the test might have passed before if none of the spawns were executed.